src/components/wallet/wallet.ts

```diff
@@ -51,6 +53,7 @@ interface Loading {
   update?: boolean
   deposit?: boolean // NEW
   withdraw?: boolean // NEW
+  claim?: boolean // NEW
```

Comment:
I think we can remove the `// NEW` comments scattered throughout the project, they don't provide any value

Reply:
Good point